apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java

```diff
@@ -345,9 +345,13 @@ public void startTest(View view) {
     }
 
     public void stopTest(View view) {
+        onStopTest();
+    }
+
+    @Override
+    public void onStopTest() {
         mUpdateThread.stop();
```

Comment:
we should still stop and close audio right?

Reply:
Yep!
super.onStopTest() calls stopAudio() and closeAudio().
Thanks for asking.